Makefile

```diff
@@ -147,10 +147,13 @@ controller-gen:
 # SRC_PKG is the path of code files
 SRC_PKG := github.com/openebs/zfs-localpv/pkg
 
+# GEN_SRC is path for all api definitions that need to be generated
+GEN_SRC := openebs.io/zfs/v1
+
 # code generation for custom resources
 .PHONY: kubegen
 kubegen: kubegendelete deepcopy-install clientset-install lister-install informer-install
-	@GEN_SRC=openebs.io/zfs/v1 make deepcopy clientset lister informer
```

Comment:
@vaniisgh all the yamls are autogenerated, so if you are adding any comments in the api file, the comment will added as description. What you need to do is run `make manifests` command. This will generate the yamls (deploy/ directory). Just add those files and push them in this PR.